### PR TITLE
add suspend field to printcolumn

### LINKED ISSUE #399: JobSet should display whether it is suspended or not.
Related: https://github.com/kubernetes/kubernetes/issues/123221

It would be nice to have a boolean value for suspend when you do `kubectl get jobset`.

### PR DESCRIPTION
**Pull request by kannon92**
Fixes #399

### CODE REVIEW

**Comment by kannon92** [comment]
Maybe https://github.com/kubernetes/test-infra/issues/31889 is related to these failures...

**Comment by danielvegamyhre** [comment]
I think this would be useful to have. Looks like the current implementation results in a nil dereference error, though.

**Comment by kannon92** [comment]
Are you bringing up the seg fault?

I am getting the same failure in the startup policy. I tested this and it worked with specifiy suspend: true, suspend:false and not specifying it.

**Comment by kannon92** [comment]
I had a feeling something is up with the CI because I am getting the same error in https://github.com/kubernetes-sigs/jobset/pull/246.

File: api/jobset/v1alpha2/jobset_types.go [hunk 1]
```diff
@@ -116,6 +116,7 @@ type ReplicatedJobStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Restarts",JSONPath=".status.restarts",type=string,description="Number of restarts"
 // +kubebuilder:printcolumn:name="Completed",type="string",priority=0,JSONPath=".status.conditions[?(@.type==\"Completed\")].status"
+// +kubebuilder:printcolumn:name="Suspended",type="string",JSONPath=".spec.suspend",description="Is this JobSet suspended"
```
Comment (ahg-g) [inline api/jobset/v1alpha2/jobset_types.go:119]:
```suggestion
// +kubebuilder:printcolumn:name="Suspended",type="string",JSONPath=".spec.suspend",description="JobSet suspended"
```

**Review by ahg-g — COMMENTED** [review]
Thanks Kevin, one suggestion.

File: api/jobset/v1alpha2/jobset_types.go (hunk 1 above)
Comment (kannon92) [reply api/jobset/v1alpha2/jobset_types.go]: updated.

**Pull request merged**